homeassistant/components/loqed/sensor.py

```diff
@@ -0,0 +1,71 @@
+"""Creates LOQED sensors."""
+from typing import Final
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    EntityCategory,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import LoqedDataCoordinator, StatusMessage
+from .entity import LoqedEntity
+
+SENSORS: Final[tuple[SensorEntityDescription, ...]] = (
+    SensorEntityDescription(
+        key="ble_strength",
+        translation_key="ble_strength",
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
```

Comment:
As this already has a device class, do we even need translations?

As in? Without a translation string, it will use the default: "Signal strength"

Reply:
The device uses both WiFi and BLE for communication. I think it's useful to make it clear that this is about the BLE strength and not the WiFi signal. Adding the WiFi signal as sensor would make that clear, but it has only limited value compared to the ble strength

Reply:
Makes sense 👍